tensorflow/core/grappler/optimizers/auto_mixed_precision_test.cc

```diff
@@ -1036,6 +1042,37 @@ int GetCudaVersion(const Cluster& cluster) {
   return 0;
 }
 
+std::pair<int, int> GetDeviceGPUArch(const Cluster& cluster){
```

Comment:
can we figure out a way to avoid the copy-paste of `GetDeviceGPUArch` ?  
G will not let us upstream this code as is.

Reply:
It's actually a different function that goes through all devices. I can rename it to avoid confusion.